lib/controllers/secrets.js

```diff
@@ -0,0 +1,12 @@
+const { Router } = require('express');
+const authenticate = require('../middleware/authenticate');
+const Secret = require('../models/Secret');
+
+module.exports = Router().get('/', authenticate, async (req, res, next) => {
+  try {
+    const data = await Secret.getAll();
+    res.json(data);
+  } catch (e) {
+    next(e);
+  }
```

Comment:
I'm not sure why, but this endpoint appears to be returning a 404. Did you happen to do a npm run setup-db in heroku after adding in the secrets table?